src/components/HandoffLink/HandoffLink.test.tsx

```diff
@@ -30,15 +29,11 @@ describe('HandoffLink', () => {
 
   it('default', async () => {
     const { getByRole } = render(
-      <TestWrapper
-        initialState={{
-          user: { id: 'userId', firstName: 'Bob', lastName: 'Jones' } as User,
-        }}
-      >
+      <GqlMockedProvider<GetUserQuery>>
```

Comment:
@OzzieOrca On this test, it seems that the click is happening when the ```user``` is undefined. When I debug I see that ```user``` goes from ```undefined``` -> ```has a value``` -> ```undefined``` -> ```has a value```. Not sure the best way to address this in the test but it works fine in the app.

Reply:
Weird. It's rerendering for some reason? Idk... You forced only one test to run?

Reply:
Yeah even running just the one test it fails. You're right about it rerendering. Not sure what else to try, I even tried creating a ```cache``` with the query but that did not work as well.